R/utils.R

```diff
@@ -598,3 +598,46 @@ wait_for_nonmem.list <- function(.mod, .time_limit = 300, .interval = 5) {
 
 }
 
+
+#' Replace BBI_NULL_NUM and BBI_NULL_STR with NA_real_ or NA_character_
+#'
+#' @param .vals a numeric or character vector
+#'
+#' @importFrom purrr map
+#' @keywords internal
+set_bbi_null <- function(.vals){
+  map(.vals, function(.val){
+    if(inherits(.val, c("integer", "numeric", "character")) && (.val == BBI_NULL_NUM)){
```

Comment:
When I read https://github.com/metrumresearchgroup/bbr/pull/524#discussion_r935081383, I thought is was just talking about the `_real` and `_integer` distinction.  Should "character" be included here?

Reply:
Yes, some occurrences of `BBI_NULL_NUM` are characters beforehand (see [here](https://github.com/metrumresearchgroup/bbr/blob/main/inst/test-refs/1_summary_obj.R#L10)). Updated the docs

Reply:
> Yes, some occurrences of BBI_NULL_NUM are characters beforehand (see [here](https://github.com/metrumresearchgroup/bbr/blob/main/inst/test-refs/1_summary_obj.R#L10)).

Er, right.  My question is whether that's the intended behavior.  As I said, that wasn't how I read the initial comment I linked to.  It was just about whether to distinguish integer vs real.

As I mentioned elsewhere, my preference would be to just assign NA to it and keep whatever the current vector type is without any of these `inherit` checks.  If we do keep the `inherit` checks, then I think it's valuable to keep the character distinction.